rotkehlchen/history/events/structures/asset_movement.py

```diff
@@ -92,6 +92,10 @@ def __init__(
             extra_data=extra_data,
         )
 
+        # Note that unique_id is not stored in the database, so this is useful only when working
+        # with newly created movements. This will be None in movements loaded from the database.
+        self.unique_id = unique_id
```

Comment:
Do you only need this for bitfinex? It looks like a hack since entries for db don't have it and will cause trouble in the future

Reply:
Yes, this was only for bitfinex, although I thought it might be needed elsewhere as well.

But I've removed it again, since it should simply be put in extra_data if it needs be available. That way it will get saved to the db as well.